cmd/direnv/direnv.go

```diff
@@ -0,0 +1,196 @@
+package direnv
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/buildsafedev/bsf/cmd/styles"
+	"github.com/spf13/cobra"
+)
+
+var Direnv = &cobra.Command{
+	Use:   "direnv",
+	Short: "direnv initializes the direnv environment for the project",
+	Run: func(cmd *cobra.Command, args []string) {
+		err := generateEnvrc()
+		if err != nil {
+			fmt.Println(styles.ErrorStyle.Render("error: ", err.Error()))
+			os.Exit(1)
+		}
+		err = fetchGitignore()
+		if err != nil {
+			fmt.Println(styles.ErrorStyle.Render("error: ", err.Error()))
+			os.Exit(1)
+		}
+
+		if envVar != "" {
+			err = setDIrenv(envVar)
+			if err != nil {
+				fmt.Println(styles.ErrorStyle.Render("error: ", err.Error()))
+				os.Exit(1)
+			}
+		}
+
+		cmD := exec.Command("direnv", "allow")
```

Comment:
I'm actually not sure if we should be the ones executing `direnv allow`. I think we should let the user do it and bsf will only help with `.envrc` generation. 

Imagine cases where direnv isn't installed, etc or a user may not have wanted it to execute/load immediately.